background/update.js

```diff
@@ -53,10 +55,17 @@ var updater = {
 
     'ignoreDigest' option is set on the second manual individual update check on the manage page.
     */
-    return (ignoreDigest ? Promise.resolve() : calcStyleDigest(style))
-      .then(maybeFetchMd5)
-      .then(maybeFetchCode)
-      .then(maybeSave)
+    let pending = Promise.resolve();
```

Comment:
ugh, now it's not as obvious; see [my attempt](https://github.com/openstyles/stylus/pull/158/files) at extending this function.